src/extras/PMREMGenerator.js

```diff
@@ -611,29 +611,31 @@ ${_getEncodings()}
 #define ENVMAP_TYPE_CUBE_UV
 #include <cube_uv_reflection_fragment>
 
+vec3 getSample(float theta) {
+  vec3 axis = latitudinal ? poleAxis : cross(poleAxis, vOutputDirection);
+  if (all(equal(axis, vec3(0.0))))
+    axis = cross(vec3(0.0, 1.0, 0.0), vOutputDirection);
+  axis = normalize(axis);
+  float cosTheta = cos(theta);
+  // Rodrigues' axis-angle rotation
+  vec3 sampleDirection = vOutputDirection * cosTheta 
+    + cross(axis, vOutputDirection) * sin(theta) 
+    + axis * dot(axis, vOutputDirection) * (1.0 - cosTheta);
```

Comment:
I can't remember if we want tabs or spaces in these files, but we should probably be consistent. This always trips me up because VSCode inevitably chooses the wrong one.

Reply:
Tabs 👌